raiden/raiden_event_handler.py

```diff
@@ -598,6 +604,148 @@ def handle_contract_send_channelunlock(
                     partner=our_address,
                 )
 
+
+    @staticmethod
+    def handle_contract_send_channelunlock_light(
```

Comment:
This looks good but the code is duplicated. 

The only difference is the `store_light_client_protocol_message`, so, i think we can generalise `handle_contract_send_channelunlock_light `and `handle_contract_send_channelunlock`

Reply:
I agree, and I think I have an idea of how to to do it.

I'll generalize in next commit.